src/mc-tokens-private.h

```diff
@@ -29,23 +29,31 @@
  * Integers are represented as uint64_t in little-endian.
  *
  * CollectionsLevel1Token = HMAC(RootKey, 1)
- * ServerTokenDerivationLevel1Token = HMAC(RootKey, 2)  <- new in v2
  * ServerDataEncryptionLevel1Token = HMAC(RootKey, 3)
  *
  * EDCToken = HMAC(CollectionsLevel1Token, 1)
  * ESCToken = HMAC(CollectionsLevel1Token, 2)
- * ECCToken = HMAC(CollectionsLevel1Token, 3)  <- deprecated in v2
```

Comment:
nit - I feel like we should keep these deprecation notes

Reply:
I did add the note about ECC related tokens being used with FLE2v1, but you're right that it doesn't explicitly call it a deprecation...  Given these are private APIs I'm not too worried about that semantic distinction, but I'll clarify it since it costs nothing...